recipes/gdist/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: gdist
+  version: "1.0.3"
+
+source:
+  md5: 98547eac8e7c0b5a7a600e12ae5c9c4c
+  fn: 'gdist-1.0.3.tar.gz'
+  url: https://pypi.io/packages/source/g/gdist/gdist-1.0.3.tar.gz
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - cython
+  run:
+    - python
+    - numpy
+    - scipy
+    - cython
+
+test:
+  imports:
+    - gdist
+
+about:
+  home: http://code.google.com/p/geodesic/
```

Comment:
This should probably be https://github.com/the-virtual-brain/tvb-geodesic, because that's where the actual source code used here lives (despite it being a fork of that one).